documentation/RLC-test-quickstart.md

```diff
@@ -0,0 +1,234 @@
+# Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [How to Run Test](#how-to-run-test)
+  - [Test strucure](#test-strucure)
+  - [Fail to run tests in the first time](#fail-to-run-tests-in-the-first-time)
+  - [Running tests in playback mode](#running-tests-in-playback-mode)
+- [Writing New Tests](#writing-new-tests)
+  - [Example: Basic Azure service interaction and recording](#example-basic-azure-service-interaction-and-recording)
+
+# Overview
+
+This page is to help you write and run tests for Azure Javascript SDK. The Azure SDK test framework uses the [`test-recorder`](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/README.md) library, which in turn rests upon on a HTTP recording system ([testproxy](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy)) that enables tests dependent on network interaction to be run offline.
+
+# Prerequisites
+
+- Docker
+  - [Docker](https://www.docker.com/get-started/) is required, as the [test proxy server](https://github.com/Azure/azure-sdk-tools/tree/main/tools/test-proxy) is running in a container during testing
```

Comment:
Any command i need to execute for Docker to start the test proxy?

Reply:
No specific command we need to execute to start test proxy

Reply:
I wonder if we should just point them to this document regarding the recorder. Most of the stuff should be the same

Reply:
@joheredi  Do you mean the quick start guide or only the section `Prerequisites`?

I referred a lot for test recorder document but I think we still need a **simpler** guide compared with Harsha's one.

Reply:
I am good to have a quick get_started for writing tests for DPG.  But please make sure the steps are easy to follow and avoid the duplicate from https://github.com/Azure/azure-sdk-for-js/pull/21950#pullrequestreview-998786756 where you could reference directly.    Please note, our original goal from this spec is '10 min Get Started Experience'

Reply:
Thanks for the suggestion! I will guide users to Harsha's one for complete guide and make this one as simple as possible.